compiler/one-cmds/CMakeLists.txt

```diff
@@ -151,3 +152,21 @@ endif(NOT ENABLE_TEST)
 add_subdirectory(dummy-driver)
 add_subdirectory(tests)
 add_subdirectory(validate-onnx2circle)
+
+# Copy O1.txt file to test directory
```

Comment:
`one-cmds` tests (https://github.com/Samsung/ONE/pull/10473) will use O1.txt.

This makes a single source of O1 list, making it easier to test default option sets.